tasks/logger-file.js

```diff
@@ -0,0 +1,19 @@
+const gulp = require('gulp');
+const pathBuilder = require('../pathBuilder');
+const config = require('../config');
+const fs = require('fs');
+
+/**
+ *  `logger-file-create` Task
+ *  ---------------
+ *
+ */
+
+gulp.task('logger-file-create', () => {
+    const fullFilePath = `${pathBuilder.jsErrorLoggerSubDir}/${config.logger.file}`;
```

Comment:
You should be able to use the `gulp-file` module combined with `gulp.dest` in order to keep the code running inside the pipe streams.

```js
const file = require('gulp-file');

gulp.task('test', () => file(config.logger.file, '', { src: true })
    .pipe(gulp.dest(pathBuilder.jsErrorLoggerDir))
);
```

Reply:
@DamianMullins The `pipe` way to do this doesn't seem to work.  I'll play around with it to see if there's another way to get this working. I've tested the current code and it does work fine though.